curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting/falsy-bouncer.md

```diff
@@ -39,7 +39,16 @@ assert.deepEqual(bouncer([false, null, 0, NaN, undefined, '']), []);
 ```js
 assert.deepEqual(bouncer([null, NaN, 1, 2, undefined]), [1, 2]);
 ```
+`Orginal array should not be mutated`.
 
+```js
+assert(
+  (function () {
+  let arr = [7, "ate", "", false, null, NaN, 1, 2, undefined];
+  bouncer(arr);
+  return arr.sort().toString() == [7, "ate", "", false, null, NaN, 1, 2, undefined].sort().toString()
+  })());
```

Comment:
```suggestion
const arr = ['a', false, 0, 'Naomi'];
bouncer(arr);
assert.deepEqual(arr, ['a', false, 0, 'Naomi'])
```

I don't think we need to be quite so complex here.

Reply:
@naomi-lgbt  Also test description is not added.

Reply:
I tried the suggested change approach earlier, but it doesn't seem to work. By looking at other tests, I came to the conclusion that every code should be in assert for it to work, which is why I've written the code this way,